Sport1Player/Sport1PlayerAdapter.m

```diff
@@ -12,14 +12,22 @@
 #import "Sport1PlayerAdapter.h"
 #import <JWPlayer_iOS_SDK/JWPlayerController.h>
 #import "Sport1PlayerLivestreamAge.h"
+#import "JWPlayerViewController+Public.h"
 
 static NSString *const kTrackingInfoKey = @"tracking_info";
 static NSString *const kAgeRatingKey = @"age_rating";
 static NSString *const kPlayableItemsKey = @"playable_items";
 static NSString *const kPluginName = @"pin_validation_plugin_id";
 static int kWatershedAge = 16;
 
+@interface Sport1PlayerAdapter ()
+@property (nonatomic) Sport1PlayerLivestreamPin *livestreamPinValidation;
+@property (nonatomic) UIView * container;
```

Comment:
Please, add the `weak` modifier. I believe you can add also strong (it will not create a circular dependency), but it should be retained already as the parent view controller, so better use weak.